### PR TITLE
New documentation theme

### PR DESCRIPTION
**Pull request by aalonsog**

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,27 @@
+0 info it worked if it ends with ok
```
Comment (jcague) [inline npm-debug.log]: we don't need this file, right?

File: mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+site_name: Documentation
+site_url: http://licode.readthedocs.org
+repo_url: https://github.com/lynckia/licode
+site_description: Open Source WebRTC Communications Platform
+docs_dir: doc
+theme: mkdocs
+theme_dir: doc/custom_theme
+# extra_css: ["https://fiware.org/style/fiware_readthedocs.css"]
```
Comment (jcague) [inline mkdocs.yml]: consider removing this comment or change it 🙈

File: doc/index.md [hunk 3]
```diff
@@ -1 +1,89 @@
-#Licode
+<h1>Architecture overview</h1>
```
Comment (jcague) [inline doc/index.md]: can a .md document has html documents inside?

File: doc/custom_theme/nav.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<a id="github_button" href="https://github.com/ging/licode" class="visible-desktop"><img style="position: fixed; top: 0; right: 0; border: 0; z-index: 1031" src="https://s3.amazonaws.com/github/ribbons/forkme_right_red_aa0000.png" alt="Fork me on GitHub"></a>
+<div class="navbar navbar-default navbar-fixed-top" role="navigation">
+    <div class="container">
+
+        <!-- Collapsed navigation -->
+        <div class="navbar-header">
+            {%- if nav|length>1 or (page and (page.next_page or page.previous_page)) or config.repo_url %}
```
Comment (jcague) [inline doc/custom_theme/nav.html]: will this embedded code run within readthedocs?

File: doc/custom_theme/toc.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<div class="bs-sidebar hidden-print affix-top well" role="complementary">
+
+    <ul class="nav bs-sidenav">
+	{%- block search_button %}
```
Comment (jcague) [inline doc/custom_theme/toc.html]: same question here... is readthedocs compatible with this code?

File: npm-debug.log (hunk 1 above)
Comment (aalonsog) [inline npm-debug.log]: Of course, we don't

File: doc/index.md (hunk 3 above)
Comment (aalonsog) [inline doc/index.md]: Yes, AFAIK markdown documents allows to insert html code.

File: doc/custom_theme/nav.html (hunk 4 above)
Comment (aalonsog) [inline doc/custom_theme/nav.html]: Yes, RTD uses mkdocs site generator to serve the page

File: doc/custom_theme/toc.html (hunk 5 above)
Comment (aalonsog) [inline doc/custom_theme/toc.html]: Same answer :)

**Pull request merged**